src/components/Hamburger/Hamburger.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import { hideVisually } from 'polished';
+
+const HAMBURGER_WIDTH = '16px';
+const LINE_HEIGHT = '2px';
+
+const buttonBaseStyles = ({ theme }) => css`
+  label: hamburger-button;
+  box-sizing: content-box;
+  cursor: pointer;
+  display: inline-block;
+  padding: ${theme.spacings.mega} ${theme.spacings.byte};
+  width: ${HAMBURGER_WIDTH};
+  background: none;
+  border: 0;
+  position: relative;
+`;
+
+const layersBaseStyles = ({ theme }) => css`
+  label: hamburger-layers;
+  margin-top: calc(${LINE_HEIGHT} / -2);
+  top: 50%;
+  width: calc(${HAMBURGER_WIDTH} * 0.77);
+  transition: 0.3s cubic-bezier(0.55, 0.055, 0.675, 0.19);
+
+  &,
+  &::after,
+  &::before {
+    background-color: ${theme.colors.n900};
+    border-radius: 1px;
+    display: block;
+    height: ${LINE_HEIGHT};
+    position: absolute;
+    transition: width 0.2s ease-out 0.14s, top 0.1s 0.34s ease-in,
+      opacity 0.1s ease-in, bottom 0.1s 0.34s ease-in,
+      transform 0.3s cubic-bezier(0.55, 0.055, 0.675, 0.19);
```

Comment:
Magic numbers :eyes:

![magic](https://media.giphy.com/media/l2YWs1NexTst9YmFG/giphy.gif)

Reply:
How would you change this? 

Personally, I feel like it's okay to have these "magic" numbers here. It seems clear what they do (control the animation) and they are highly specific to this component, so there's no need to extract them.

Reply:
I'm also okay with them here, just wanted to use the gif actually.
Sorry if looks too much niptick hahaha